### ISSUE TITLE
Feature request - Add ability to run a single group and make it repeatable

### ISSUE THREAD

**Issue by abid-mujtaba** [devforth/scriptimate#9]
Full disclosure, I tried to make this work by creating a dummy group:

```text
define_group _:
    animate_100 pause
```

and then used `run_groups_together <some-group> _` which works out fairly well and allows me to use `define_group` for grouping commands together even when I do **not** want to run multiple groups in parallel.

It would be nice to have a different command, for example, `run_group <single-group>`, which does the same thing so I can avoid this hack.

Additionally, it would be nice to have the ability to be able to run the same group multiple times, I have some sub-loops that I want to run 2 or 3 times (interleaved with other stuff). This does not seem to be currently possible. Repeated invocations of `run_groups_together` seem to be ignored.
